test-tools/wamr-ide/VSCode-Extension/src/view/NewProjectPanel.ts

```diff
@@ -178,16 +186,21 @@ export class NewProjectPanel {
                             );
                             return;
                         } else if (
-                            this._creatNewProject(
-                                message.projectName,
-                                message.template,
-                                extensionUri
-                            ) === NewProjectPanel.USER_INTPUT_ERR
+                            createNewProjectStatus ===
+                            NewProjectPanel.USER_INTPUT_ERR
                         ) {
                             vscode.window.showErrorMessage(
                                 'Please fill chart before your submit!'
                             );
                             return;
+                        } else if (
+                            createNewProjectStatus ===
+                            NewProjectPanel.DIR_PATH_INVALID_ERR
+                        ) {
+                            vscode.window.showErrorMessage(
+                                `A file name can't contain any of the following charachters: / \\ : * ? < > | " and length less than 255`
```

Comment:
Can the string be wrapped by `? Note that it isn't '

Reply:
Updated